packages/storybook/src/react-components/link/visual/States.tsx

```diff
@@ -0,0 +1,22 @@
+import { LuxLink } from '@lux-design-system/components-react';
+
+export const VisualStates = () => (
+  <div className="utrecht-document">
```

Comment:
```suggestion
  <>
```

@MrSkippy Ik heb sterk het gevoel dat dit niet klopt. Mijns inziens zou je niet verplicht `utrecht-document` moeten gebruiken om mooie links te hebben. Wat jij?

Reply:
Bij de links wel, helaas. Er is geen font-token voor de link, het `inherit` van de parent component, vandaar.